### PR TITLE
Create pt2_bug_report.yml

### PR DESCRIPTION
**Pull request by ngimel**
Moves pt2 bug template from dynamo, we want all user issues to be filed in pytorch/pytorch repo

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/pt2_bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: 🐛 torch.compile Bug Report
+description: Create a report to help us reproduce and fix the bug
+labels: ["bug"]
```
Comment (ngimel) [inline .github/ISSUE_TEMPLATE/pt2_bug_report.yml:3]:
```suggestion
labels: ["oncall: pt2"]
```

File: .github/ISSUE_TEMPLATE/pt2_bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        Please provide the error you're seeing
+      placeholder: |
+        Error...
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Minified repro
+      description: |
+        Please run the minifier on your example and paste the minified code below
```
Comment (ngimel) [inline .github/ISSUE_TEMPLATE/pt2_bug_report.yml:35]:
```suggestion
        Please run the minifier on your example and paste the minified code below
```

File: .github/ISSUE_TEMPLATE/pt2_bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        env TORCHDYNAMO_REPRO_AFTER="dynamo" python your_model.py
+
+        import torch
+        ... 
+
+        # torch version: 1.13.....
+
+        class Repro(torch.nn.Module)
+    validations:
+      required: false
```
Comment (albanD) [inline .github/ISSUE_TEMPLATE/pt2_bug_report.yml:49]: Do you want to add the same section as the regular one for getting version through the collect_env script? https://github.com/pytorch/pytorch/blame/9c6433ce48c27f4b6ecf585bf6049176c217bbbe/.github/ISSUE_TEMPLATE/bug-report.yml#L41-L52

Also should we collect pt2 related info in that script (like compiler, triton version, nvcc available, etc)?

**Review by albanD — APPROVED** [review]
Sounds good.
Optional suggestion about collecting environment to ease repro

File: .github/ISSUE_TEMPLATE/pt2_bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        Learn more here https://pytorch.org/docs/master/dynamo/troubleshooting.html
+      placeholder: |
+        env TORCHDYNAMO_REPRO_AFTER="aot" python your_model.py
+        or
+        env TORCHDYNAMO_REPRO_AFTER="dynamo" python your_model.py
+
+        import torch
+        ... 
+
+        # torch version: 1.13.....
```
Comment (albanD) [inline .github/ISSUE_TEMPLATE/pt2_bug_report.yml:45]: 2.0 ?

File: .github/ISSUE_TEMPLATE/pt2_bug_report.yml (hunk 3 above)
Comment (ngimel) [reply .github/ISSUE_TEMPLATE/pt2_bug_report.yml:49]: Yeah good idea

**Pull request closed without merging**